### ISSUE TITLE
Importing @aws-sdk/client-glacier throws "Error: Cannot find module '@aws-sdk/body-checksum-node'"

### ISSUE THREAD

**Issue by trivikr** [aws/aws-sdk-js-v3#849]
**Describe the bug**
Importing @aws-sdk/client-glacier throws "Error: Cannot find module '@aws-sdk/body-checksum-node'"

**SDK version number**
```console
$ yarn list @aws-sdk/client-glacier
└─ @aws-sdk/client-glacier@1.0.0-alpha.12
```

**Is the issue in the browser/Node.js?**
Node.js

**Details of the browser/Node.js version**
v10.18.0

**To Reproduce (observed behavior)**
```console
$ yarn add @aws-sdk/client-glacier@alpha
$ node
> const { Glacier } = require("@aws-sdk/client-glacier");
Thrown:
{ Error: Cannot find module '@aws-sdk/body-checksum-node'
    at Function.Module._resolveFilename (internal/modules/cjs/loader.js:636:15)
    at Function.Module._load (internal/modules/cjs/loader.js:562:25)
    at Module.require (internal/modules/cjs/loader.js:692:17)
    at require (internal/modules/cjs/helpers.js:25:18) code: 'MODULE_NOT_FOUND' }
```

**Expected behavior**
The import should not throw any error

**Comment by srchase** [comment]
The generated Glacier client includes these two lines in the package.json:

```
    "@aws-sdk/body-checksum-browser": "^1.0.0-alpha.0",
    "@aws-sdk/body-checksum-node": "^1.0.0-alpha.0",
```

Those dependencies are missing here: https://github.com/aws/aws-sdk-js-v3/blob/smithy-codegen/clients/client-glacier/package.json

**Comment by srchase** [comment]
May be same issue as https://github.com/aws/aws-sdk-js-v3/issues/804

**Comment by trivikr** [comment]
The dependencies were not manually added in `clients/client-glacier/GlacierClient.ts` in PR https://github.com/aws/aws-sdk-js-v3/pull/789
* The CI for the PR didn't fail because of issue discussed in https://github.com/aws/aws-sdk-js-v3/issues/817

The dependencies were not added by CodeGen because of issue discussed in https://github.com/aws/aws-sdk-js-v3/issues/804

**Comment by trivikr** [comment]
The release script caught missing dependencies in `body-checksum-*` packages (fixed in https://github.com/aws/aws-sdk-js-v3/pull/839), but not in `client-glacier`

**Comment by trivikr** [comment]
Verified that the issue is resolved in `@aws-sdk/client-glacier@1.0.0-alpha.13`
